isis/appdata/translations/SerialNumber0001.trn

```diff
@@ -0,0 +1,24 @@
+Group = Keyword1
+  Auto
+  OutputPosition = ("Group","SerialNumberKeywords")
+  OutputName = Keyword1
+  InputGroup = "IsisCube,Instrument"
+  InputKey = SpacecraftName
+  Translation = (*,*)
+EndGroup
+Group = Keyword2
+  Auto
+  OutputPosition = ("Group","SerialNumberKeywords")
+  OutputName = Keyword2
+  InputGroup = "IsisCube,Instrument"
+  InputKey = InstrumentId
+  Translation = (*,*)
+EndGroup
+Group = Keyword3
+  Auto
+  OutputPosition = ("Group","SerialNumberKeywords")
+  OutputName = Keyword3
+  InputGroup = "IsisCube,Instrument"
+  InputKey = SpacecraftClockCount
+  Translation = (*,*)
+EndGroup
```

Comment:
For my own education, what is this one (0001 or 0002) used for?

Reply:
I just caught that I missed this, so I renamed and put in the serial numbers directory. I can't find where this is actually used! It may not be.